src/main/java/com/matt/forgehax/mods/MatrixNotifications.java

```diff
@@ -224,13 +242,22 @@ private static String getUriUuid() {
   
   private void notify(String message) {
     JsonObject object = new JsonObject();
-    object.addProperty("text", message);
-    object.addProperty("format", "plain");
-    object.addProperty("displayName", MC.getSession().getUsername());
+	if (provider.get().equals("Discord")) {
+    	object.addProperty("content", message);
```

Comment:
😭 tabbing pls

Reply:
wow I don't know what happened here, sorry it slipped!